### PR TITLE
PR: Fix missing spaces in various docstrings.

### PR DESCRIPTION
**Pull request by pavithraes**
Fixes issue: #505

### CODE REVIEW

File: colour/models/rgb/transfer_functions/__init__.py [hunk 1]
```diff
@@ -777,9 +777,9 @@ def encoding_cctf(value, function='sRGB', **kwargs):
 
     if 'itu-r bt.2100' in function.lower():
         usage_warning(
-            'With the "ITU-R BT.2100" method, only the inverse electro-optical'
+            'With the ITU-R BT.2100 method, only the inverse electro-optical '
```
Comment (pavithraes) [inline colour/models/rgb/transfer_functions/__init__.py]: On adding a space, some checks failed with the following message:

```
E501 line too long (80 > 79 characters)
            'With the "ITU-R BT.2100" method, only the inverse electro-optical '
                                                                               ^
```

To fix this, I removed the double quotes around ITU-R BT.2100 at both places.

Comment (KelSolaar) [reply colour/models/rgb/transfer_functions/__init__.py]: We should keep them ideally for emphasis, is it possible to actually wrap the lines please?

Cheers,

Comment (pavithraes) [reply colour/models/rgb/transfer_functions/__init__.py]: Sure! It's done. Do the current changes work?

Comment (KelSolaar) [reply colour/models/rgb/transfer_functions/__init__.py]: Yes thank you! :)

**Comment by KelSolaar** [comment]
Thanks @pavithraes ! 👍

**Pull request merged**